clients/src/main/java/org/apache/kafka/common/metrics/Sensor.java

```diff
@@ -133,6 +135,10 @@ public String name() {
         return this.name;
     }
 
+    List<Sensor> parents() {
```

Comment:
Can we add package-private methods without a KIP?

Reply:
This should be fine.